src/services/Database.ts

```diff
@@ -262,8 +261,6 @@ export default class Database {
       await this.put("Ids", {type: "WORKER1_ONE_SIGNAL_SW_VERSION", id: state.workerVersion});
     if (state.updaterWorkerVersion)
       await this.put("Ids", {type: "WORKER2_ONE_SIGNAL_SW_VERSION", id: state.updaterWorkerVersion});
-    if (state.backupNotification)
-      await this.put("Ids", {type: "backupNotification", id: state.backupNotification});
```

Comment:
Nit: there should be a space after and before brackets

Reply:
same for above calls

Reply:
👍